src/Serilog.Sinks.Elasticsearch/LoggerConfigurationElasticSearchExtensions.cs

```diff
@@ -170,6 +171,7 @@ public static class LoggerConfigurationElasticsearchExtensions
             string deadLetterIndexName = null,
             int? numberOfShards = null,
             int? numberOfReplicas = null,
+            string[] indexAliases = null,
```

Comment:
Adding a new parameter is a breaking change as it changes the signature. Do you really need this here and not just as a property on the options object?

Reply:
Removed from there. You are right, its acceptable to assign value later, not from that extension method.